sponsors/templates/sponsors/sponsors.html

```diff
@@ -0,0 +1,40 @@
+{% load sponsors %}
+{% sponsors as all_sponsors %}
+{% packages as all_packages %}
+
+<h1>Here are the list of sponsors!</h1>
+<ul>
+    {% for sponsor in all_sponsors.sponsors %}
+        <li>Name: {{ sponsor.name }}</li>
+        <li>Description: {{ sponsor.description }}</li>
+        <li>Url: {{ sponsor.url }}</li>
+        <ul>
+            {% for package in sponsor.packages.all %}
+                <li>{{ package.name }} ({{ package.currency }}{{ package.amount }})</li>
+            {% endfor %}
+        </ul>
+        {% sponsor_tagged_image sponsor "main_logo" as logo_url %}
+        {% if logo_url %}
+            <img width="30%" height="30%" src="{{ logo_url }}" alt="{{ sponsor.name }}" title="{{ sponsor.name }}" />
+        {% else %}
+            <span class="nologo">{{ sponsor.name }}</span>
+        {% endif %}
+        <br />
+        ---------
+    {% empty %}
+        <li>No sponsors</li>
+    {% endfor %}
+</ul>
```

Comment:
I was hoping we could display our sponsors in this format, a row for each sponsor and put a "We are hiring" label if they are hiring. Like we did here https://2023.djangocon.africa/our-sponsors/ but just do this dynamically. Also group the sponsors by package/level.

Reply:
This is just a boilerplate template, we can update this after sponsor template design is done. :slightly_smiling_face: